samples/apps/copilot-chat-app/WebApp/src/libs/useChat.ts

```diff
@@ -59,14 +63,16 @@ export const useChat = () => {
         const ask = { input: value, variables: [{ key: 'audience', value: account?.name ?? 'Unknown User' }] };
         try {
             var result = await sk.invokeAsync(ask, 'ChatSkill', 'Chat');
+            // var result = await connectors.invokeSkillWithGraphToken(ask, 'ChatSkill', 'Chat');
```

Comment:
Why do we need the graph token for Chat? Is this just an example statement?

Reply:
Yeah this is just an example, I can change the skill to OpenAI Skill to make it more clear

Reply:
Changed to be more generic to ConnectorSkill. Added some helpful sections in ReadMe as well